tests/test_all.py

```diff
@@ -59,6 +61,19 @@ def test_os_release(auto_container):
         ("VERSION_ID", OS_VERSION),
         ("PRETTY_NAME", OS_PRETTY_NAME),
     ):
+        if OS_VERSION == "latest" and var_name == "VERSION_ID":
+            # on openSUSE Tumbleweed that is the an ever changing snapshot date
+            # just check whether it starts with current year
+            assert (
+                int(
+                    auto_container.connection.run_expect(
+                        [0], f". /etc/os-release && echo ${var_name}"
+                    ).stdout.strip()[:4]
+                )
+                == datetime.datetime.now().year
+            )
```

Comment:
I'm pretty certain this test will fail on January 1st :wink:

Reply:
Sure, but I'm pretty certain that noone will care ;)

Reply:
I don't disagree with you on this point :wink: